cves/kernel/CVE-2017-5548.yml

```diff
@@ -450,7 +475,10 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: | 
+    Failing to properly manage memory in the buffer. This can allow people to exploit 
```

Comment:
Consider adding a designation wether this was caused primarily by a slip, lapse or planning error.